torch/onnx/symbolic_opset11.py

```diff
@@ -487,7 +487,17 @@ def _get_arange_dtype(dtype):
         dtype = sym_help._maybe_get_const(dtype, 'i')
         return dtype
 
-    if len(args) == 5:
+    if len(args) == 2:
```

Comment:
If the logic in len(args)==2 branch is very close to logic in len(args)==5 branch, can you see if you can refactor the code to remove duplicate lines?
Same for the other branches that can be merged.

Reply:
Made necessary refactoring changes